File: AssignmentEcommerce_Backend/Models/Review.cs
using System;

namespace AssignmentEcommerce_Backend.Models
{
    public class Review
    {
        public string ReviewId { get; set; }

        public string Content { get; set; }

        public int Rating { get; set; }

        public string ProductId { get; set; }

        public string UserId { get; set; }

        public string UserName { get; set; }

        public DateTime DateReview { get; set; }

        public virtual Product Product { get; set; }
    }
}

File: angular-ec/src/models/review.ts
export class Review {
  reviewId: string;
  content: string;
  rating: number;
  productId: string;
  userId: string;
  userName: string;
  dateReview: Date;

  constructor(
    reviewId: string,
    content: string,
    rating: number,
    productId: string,
    userId: string,
    userName: string,
    dateReview: Date
  ) {
    this.reviewId = reviewId;
    this.content = content;
    this.rating = rating;
    this.productId = productId;
    this.userId = userId;
    this.userName = userName;
    this.dateReview = dateReview;
  }
}
